dtde/spark/cli/cluster_create.py

```diff
@@ -40,7 +47,7 @@ def execute(args: typing.NamedTuple):
     log.info("wait for cluster:        %s", args.wait)
     log.info("username:                %s", args.username)
     if args.password:
-        log.info("Password: %s", '*' * len(args.password))
+        log.info("Password: %s", '*****')
```

Comment:
cool meant to do that

Reply:
nitpick - can we add some spaces after the 'Password:' str and before '%s' so that it lines up with the rest of the lines.